configure.ac

```diff
@@ -1286,6 +1286,7 @@ UL_BUILD_INIT([libmount])
 UL_REQUIRES_BUILD([libmount], [libblkid])
 UL_REQUIRES_HAVE([libmount], [dirfd,ddfd], [dirfd or ddfd function])
 UL_REQUIRES_HAVE([libmount], [mntent_h], [mntent.h header file])
+UL_REQUIRES_HAVE([libmount], [sys_vfs_h], [sys/vfs.h header file])
```

Comment:
Not tested, right? ;-)

We also need `have_sys_vfs_h=$ac_cv_header_sys_vfs_h` somewhere around line ~500.